<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopoChroniaCreateNodeGridDialog</class>
 <widget class="QDialog" name="TopoChroniaCreateNodeGridDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TopoChronia Phase I: Create Node Grid</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>452</width>
     <height>340</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QListWidget" name="age_listWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="Action_CreateAllAgeList_pushButton">
          <property name="text">
           <string>01 - Create Age List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Action_FeatureConversion_pushButton">
          <property name="text">
           <string>02 - Convert Features</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Action_MergeNodes_pushButton">
          <property name="text">
           <string>03 - Merge All Nodes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Action_CleanNodes_pushButton">
          <property name="text">
           <string>04 - Clean Nodes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Action_ProcessEverything_pushButton">
          <property name="styleSheet">
           <string notr="true">background-color: #5DAE8B;
color: white;</string>
          </property>
          <property name="text">
           <string>Directly Process All Steps</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="button_box">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>TopoChroniaCreateNodeGridDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>TopoChroniaCreateNodeGridDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
